src/main/java/org/stellar/sdk/TransactionBuilder.java

```diff
@@ -20,6 +22,7 @@ public class TransactionBuilder {
   private Integer mBaseFee;
   private Network mNetwork;
   private TransactionPreconditions mPreconditions;
+  private Optional<SorobanTransactionData> mSorobanData;
```

Comment:
same [recommendation](https://github.com/stellar/java-stellar-sdk/pull/481/files#r1270983434) here to not use Optional

Reply:
Fixed in 37e0e19accdadde748a715af4e5dbf25a73ad40e